Gemfile.lock

```diff
@@ -35,44 +35,44 @@ GEM
   specs:
     aasm (5.1.1)
       concurrent-ruby (~> 1.0)
-    actioncable (5.2.4.5)
-      actionpack (= 5.2.4.5)
+    actioncable (5.2.5)
+      actionpack (= 5.2.5)
```

Comment:
I did a bundler update as I was having mimemagic issues. It seemed to get rid of the dependency altogether.

Reply:
Yep, they updated rails/marcel to remove it entirely. Which should make things simpler for us in future.